### PR TITLE
feat(filter): accept filter

### PR DESCRIPTION
**Pull request by Ryan-Git**

### CODE REVIEW

File: docs/2.0/05-filters-en.md [hunk 1]
```diff
@@ -37,6 +38,20 @@ match-dml-op = "delete"
 
 For the above configuration, all the `delete` `DML` message with `schema` set to `test` are rejected.
 
+## `accept` configuration
+
+The `reject` Filter rejects specific Input messages. Then these messages will not be the next Filter or Output.
```
Comment (ming535) [inline docs/2.0/05-filters-en.md]: The `accept` is a whitelist filter that accepts...

File: docs/2.0/05-filters-en.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Filter defines the column mutation operations on the Input message. It is config
 Currently, DRC supports the following Filter plugins:
 
 - `reject`: Ignores specific source messages.
```
Comment (ming535) [inline docs/2.0/05-filters-en.md]: Ignores specific source messages. It's a blacklist filter that rejects specific messages.

**Review by ming535 — CHANGES_REQUESTED** [review]

File: docs/2.0/05-filters-en.md (hunk 1 above)
Comment (Ryan-Git) [reply docs/2.0/05-filters-en.md]: done

File: docs/2.0/05-filters-en.md (hunk 2 above)
Comment (Ryan-Git) [reply docs/2.0/05-filters-en.md]: done

**Review by ming535 — APPROVED** [review]

**Pull request merged**